packages/sync-actions/test/cart-discounts-sync.spec.js

```diff
@@ -100,8 +100,8 @@ describe('Cart Discounts Exports', () => {
         expect.arrayContaining([
           {
             action: 'setKey',
-            key: 'key'
-          }
+            key: 'key',
```

Comment:
hmm these formatting changes shouldnt be here, could you please run `yarn format` inside the root folder to fix them?

Reply:
i run `yarn format` inside the root folder, but i got them still, i dont know why i got them